day-range-picker/styles/day-range-picker.css

```diff
@@ -0,0 +1,282 @@
+@charset "UTF-8";
+
+.daypicker-range {
+  --daypicker-primary-color: #ffffff;
+  --daypicker-inactive-color: #999;
+  --daypicker-active-color: #E3E200;
+  --daypicker-border: #fff;
+  --bg-daypicker-inactive-color: #2E5162;
+  --bg-daypicker: #253846;
+  z-index: 9999;
+  display: block;
+  position: relative;
+  color: var(--daypicker-primary-color); /*COLOR: #2E5162*/
+  background: var(--bg-daypicker);
+  border: 1px solid #ccc;
+  border-bottom-color: #bbb;
+  font-family: "Helvetica Neue", Helvetica, Arial, sans-serif;
+}
+
+.days-name {
+  height: 32px;
+}
+/* 
+th {
+  width: 65px;
+} */
+
+.daypicker-range:before,
+.daypicker-range:after {
+  content: " ";
+  display: table;
+}
+.daypicker-range:after {
+  clear: both;
+}
+
+.daypicker-range.is-hidden {
+  display: none;
+}
+
+.daypicker-range.is-bound {
+  position: absolute;
+  box-shadow: 0 5px 15px -5px rgba(0, 0, 0, 0.5);
+}
+
+.daypicker-lendar {
+  float: left;
+  width: 324px; /*TODO: set width here*/
+  margin: 8px;
+}
+
+@media (max-width: 767px) {
+  .daypicker-lendar {
+    float: left;
+    width: 320px; /*TODO: set width here*/
+    margin: 8px;
+  }
+}
+
+.daypicker-title {
+  position: relative;
+  text-align: center;
+  padding: 16px 0;
+  border-bottom: 1px solid var(--daypicker-primary-color);  /*set title border*/
+}
+
+.daypicker-label {
+  display: inline-block;
+  position: relative;
+  z-index: 9999;
+  overflow: hidden;
+  margin: 0;
+  padding: 5px 3px;
+  font-size: 20px;
+  line-height: 24px;
+  font-weight: bold;
+  background-color: var(--bg-daypicker-range);
+}
+.daypicker-title select {
+  cursor: pointer;
+  position: absolute;
+  z-index: 9998;
+  margin: 0;
+  left: 0;
+  top: 5px;
+  opacity: 0;
+}
+
+.daypicker-prev,
+.daypicker-next {
+  display: block;
+  cursor: pointer;
+  position: relative;
+  outline: none;
+  border: 0;
+  padding: 0;
+  width: 20px;
+  height: 30px;
+  /* hide text using text-indent trick, using width value (it's enough) */
+  text-indent: 20px;
+  white-space: nowrap;
+  overflow: hidden;
+  background-color: transparent;
+  background-position: center center;
+  background-repeat: no-repeat;
+  background-size: 10 20;
+  opacity: 0.5;
+}
+
+.daypicker-prev:hover,
+.daypicker-next:hover {
+  opacity: 1;
+}
+
+.daypicker-prev {
+  float: left;
+  background: url("data:image/svg+xml;utf8,<svg xmlns='http://www.w3.org/2000/svg' viewBox='0 0 13 22' width='13' height='22'  fill='none'><g><path d='M12 21L2 11L12 1' stroke='red' stroke-width='2'/></g></svg>") no-repeat;
```

Comment:
we should make the background images for the arrows CSS variables + make sure that they are the right color as determined by the right CSS variable